### PR TITLE
get tests on data package running

### PR DESCRIPTION
**Pull request by dglaetzer**

### CODE REVIEW

File: lib/galaxy/util/tool_shed/common_util.py [hunk 1]
```diff
@@ -2,12 +2,13 @@
 import logging
 import os
 
+from routes import url_for
+
```
Comment (nsoranzo) [inline lib/galaxy/util/tool_shed/common_util.py]: Can you remove the empty line to make the flake8 Python linter happy, please?

**Comment by martenson** [comment]
@dglaetzer  This branch is now in conflict and needs to be rebased / merged with upstream, let me know if you want help with that.

**Comment by jmchilton** [comment]
@galaxybot test this

**Comment by jmchilton** [comment]
@galaxybot test this

**Comment by martenson** [comment]
@galaxybot test this

**Pull request merged**